src/main/java/de/chojo/repbot/config/Configuration.java

```diff
@@ -100,4 +100,20 @@ public void setExclusiveHelp(boolean exclusiveHelp) {
     public boolean canEqual(Object other) {
         return configFile.canEqual(other);
     }
+    
+    public String getMagicImageLink() {
+        return configFile.getMagicImage().getMagicImageLink();
+    }
+    
+    public int getMagicImagineChance() {
+        return configFile.getMagicImage().getMagicImagineChance();
+    }
+    
+    public int getMagicImageCooldown() {
+        return configFile.getMagicImage().getMagicImageCooldown();
+    }
+    
+    public int getMagicImageDeleteSchedule() {
+        return configFile.getMagicImage().getMagicImageDeleteSchedule();
+    }
```

Comment:
please only delegate to the getMagicImage object and unwrap the values where you need them.